step-capstone/src/Components/AddFlight.js

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect } from 'react'
+import { useState } from 'react'
+import {
+    TextField,
+    Grid,
+    Checkbox
+} from '@material-ui/core';
+import {
+    MuiPickersUtilsProvider,
+    DateTimePicker
+} from '@material-ui/pickers';
+import DateFnsUtils from "@date-io/date-fns";
+
+export default function AddFlight(props) {
+    const [departureDate, handleStartChange] = useState(new Date());
+    const [arrivalDate, handleEndChange] = useState(new Date());
+    const [checked, setChecked] = useState(false);
+    const [departureAirport, setDepartureAirport] = useState("");
```

Comment:
Something to think about: Do you care about whether these are airport codes or place names? Do you want to mention that in the form or validate it somewhere?